ports/esp8266/README.md

```diff
@@ -146,7 +165,7 @@ Documentation
 -------------
 
 More detailed documentation and instructions can be found at
-http://docs.micropython.org/en/latest/esp8266/ , which includes Quick
+http://docs.micropython.org/en/latest/esp8266/, which includes Quick
```

Comment:
Isn't the space necessary to prevent markdown from including the `,` in the URL?

Reply:
Ah makes sense. Will fix. Thanks

Reply:
fixed during merge